xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -394,10 +394,13 @@ tsl::Status CudnnSupport::Init() {
   ScopedActivateExecutorContext context(parent_);
 
   // Peek at the last error to give more information in cases of errors.
-  cudaError_t cerr = cudaPeekAtLastError();
-  if (cerr != cudaSuccess) {
-    LOG(WARNING) << "There was an error before creating cudnn handle: "
-                 << cudaGetErrorName(cerr) << " : " << cudaGetErrorString(cerr);
+  cudaError_t cuda_error = cudaPeekAtLastError();
+  if (cuda_error != cudaSuccess) {
+    // Printing the cuda_error value is useful when cudaGetErrorName doesn't work.
+    LOG(WARNING) << "There was an error before creating cudnn handle ("
+                 << cuda_error << "): " << cudaGetErrorName(cuda_error) << " : "
+                 << cudaGetErrorString(cuda_error);
+    return tsl::Status(absl::StatusCode::kInternal, error);
```

Comment:
`error` is undefined

Reply:
I fixed that and added another place with an equivalent error detection mechanism.